modules/apps/josh-lapurd-theme/src/WEB-INF/src/resources-importer/sitemap.json

```diff
@@ -0,0 +1,120 @@
+{
+	"layoutTemplateId": "1_2_1_columns",
+	"publicPages": [
+		{
+			"columns": [
+				[
+					{
+						"portletId": "com_liferay_journal_content_web_portlet_JournalContentPortlet",
+						"portletPreferences" : {
+							"articleId": "img_Hero.xml",
+							"groupId": "${groupId}",
+							"portletSetupPortletDecoratorId": "barebone"
+						}
+					},
+					{
+						"portletId": "com_liferay_journal_content_web_portlet_JournalContentPortlet",
+						"portletPreferences" : {
+							"articleId": "home_Page_Site_Intro.xml",
+							"groupId": "${groupId}",
+							"portletSetupPortletDecoratorId": "barebone"
+						}
+					}
+				],
+				[
+					{
+						"portletId": "com_liferay_journal_content_web_portlet_JournalContentPortlet",
+						"portletPreferences": {
+							"articleId": "Home_Page_Grid_Left_1.xml",
```

Comment:
Rename article id's to all lower case since we have the underscore to separate the words.

Ex.
```
home_page_grid_left_1.xml
```